phys/module_surface_driver.F

```diff
@@ -176,7 +175,7 @@ SUBROUTINE surface_driver(                                         &
      &          ,flxhumr_urb2d,flxhumb_urb2d,flxhumg_urb2d            & !H urban
      &          ,trl_urb3d,tbl_urb3d,tgl_urb3d                        & !H urban
      &          ,sh_urb2d,lh_urb2d,g_urb2d,rn_urb2d,ts_urb2d          & !H urban
-     &          ,frc_urb2d, utype_urb2d                               & !H urban
+     &          ,frc_urb2d, utype_urb2d,swddir,swddif                 & !H urban
```

Comment:
This causes a failure in 
```
module_PHYSICS_CALLS.f90:1399:26:

 &y_interval,fasdas=fasdas)
                          1
Error: Missing actual argument for argument 'swddir' at (1)
```
These variables could be declared optional, because there is an explicit interface for each variable in the calling routine.